src/key.js

```diff
@@ -11,7 +11,7 @@ var h = syn.helpers,
 		var result;
 
 		try {
-			result = el.selectionStart !== undefined;
+			result = ((el.selectionStart !== undefined) && (el.selectionStart !== null));
```

Comment:
Just FYI, you didn’t have to include the parenthesis here; this would have been sufficient: `result = el.selectionStart !== undefined && el.selectionStart !== null;`